src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -232,7 +232,6 @@
           error]
          [rn/touchable-opacity
           {:hit-slop       {:top 6 :bottom 20 :left 0 :right 0}
-           :style          {:margin-left -4}
```

Comment:
I removed this `:margin-left -4` to create the space between the words,
@ajayesivan it seems you added it, I removed it, but I was wondering why do we had it before, do you remember about it? :) thanks!

Reply:
I don't remember why I added it, but at that time it looked right.

Reply:
Thank you so much!
 I was afraid I could be breaking something! :+1: